docs/developer/taxes.mdx

```diff
@@ -0,0 +1,356 @@
+          slug
+        }
+        chargeTaxes
+        displayGrossPrices
+        pricesEnteredWithTax
+        taxCalculationStrategy
+      }
+    }
+  }
+}
+```
+
+To query a specific tax configuration by ID, use the query below (in this case, we’re only asking for the `taxCalculationStrategy` field):
+
+```graphql
+query TaxConfiguration {
+  taxConfiguration(id: "VGF4Q29uZmlndXJhdGlvbjox") {
+    taxCalculationStrategy
+  }
+}
+```
+
+### Updating the tax configuration
+
+The `taxConfigurationUpdate` mutation allows you to update the specific tax configuration. The example below shows how to set the tax calculation strategy to Dynamic Taxes (in API represented as `TAX_APP` option):
+
+```graphql
+mutation {
+  taxConfigurationUpdate(
+    id: "VGF4Q29uZmlndXJhdGlvbjox"
+    input: { taxCalculationStrategy: TAX_APP }
+  ) {
+    errors {
+      field
+      message
+    }
+    taxConfiguration {
+      taxCalculationStrategy
+    }
+  }
+}
+```
+
+### Overriding tax configuration per country
+
+Tax configurations specific to a channel can be modified for different countries. For example, a single sales channel may use Flat Rates for EU countries, but use the tax app for the US. To represent this scenario in Saleor, you would set Flat Rates as the channel's default tax calculation strategy and override the configuration for the US to use Dynamic Taxes.
+
+```graphql
+mutation {
+  taxConfigurationUpdate(
+    id: "VGF4Q29uZmlndXJhdGlvbjox"
+    input: {
+      taxCalculationStrategy: FLAT_RATES
+      updateCountriesConfiguration: [
+        {
+          countryCode: US
+          taxCalculationStrategy: TAX_APP
+          displayGrossPrices: false
+          chargeTaxes: true
+        }
+      ]
+    }
+  ) {
+    errors {
+      field
+      message
+    }
+    taxConfiguration {
+      taxCalculationStrategy
+      countries {
+        country {
+          code
+        }
+        chargeTaxes
+        taxCalculationStrategy
+        displayGrossPrices
+      }
+    }
+  }
+}
+```
+
+To remove the country-specific configuration, use the `removeCountriesConfiguration` field and provide the country code for which to remove the configuration:
+
+```graphql
+mutation {
+  taxConfigurationUpdate(
+    id: "VGF4Q29uZmlndXJhdGlvbjox"
+    input: { removeCountriesConfiguration: [US] }
+  ) {
+    errors {
+      field
+      message
+    }
+    taxConfiguration {
+      countries {
+        country {
+          code
+        }
+        taxCalculationStrategy
+      }
+    }
+  }
+}
+```
+
+## Flat Rates
+
+Flat Rates are the default tax calculation strategy that is enabled for any new sales channel. With this method, you can configure static tax rates and associate them with products, product types, or shipping methods. Tax rates can be defined either as default country rates or can be overridden for specific products with tax classes. During checkout, Saleor uses either default country rates or overridden values from tax classes.
+
```

Comment:
```suggestion
## Flat rates

Flat rates are the default tax calculation strategy that is enabled for any new sales channel. With this method, you can configure static tax rates and associate them with products, product types, or shipping methods. Tax rates can be defined as default country rates or overridden for specific products with tax classes.

```

The last sentence feels redundant. We can remove it or change it to something like:
"Tax rates can be defined as default country rates or overridden for specific products with tax classes. The configuration will be respected in the checkout."